### PR TITLE
update CI testing with pip upgrade

### LINKED ISSUE #2158: add pt 1.5 for Win env
## 🚀 Feature

add testing for PyTorch 1.5 on Win environment

### Additional context

https://github.com/pytorch/pytorch/issues/32186#issuecomment-643056274

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

Fix #2158 

## PR review    
Anyone in the community is free to review the PR once the tests have passed.     
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by williamFalcon** [comment]
@Borda please leave trending contributors at the top...

File: tests/models/test_tpu.py [hunk 1]
```diff
@@ -51,42 +51,24 @@ def test_model_tpu_cores_1(tmpdir):
     tpipes.run_model_test(trainer_options, model, on_gpu=False, with_hpc=False)
 
 
+@pytest.mark.parametrize('tpu_core', [1, 5])
```
Comment (awaelchli) [inline tests/models/test_tpu.py:54]: I thought this did not work (test skipped) and the test had to be split on purpose? Are you sure about it

Comment (Borda) [reply tests/models/test_tpu.py:54]: It did not work with spawn decorator... If you open the TPU test details you should see all tests passed...

Comment (Borda) [reply tests/models/test_tpu.py:54]: In fact it was failing for a while because of bad assets =)

**Pull request merged**